test/widgets/login_test.dart

```diff
@@ -113,6 +113,27 @@ void main() {
           id: testBinding.globalStore.accounts.single.id));
     });
 
+    testWidgets('account already exists', (tester) async {
+      final zulipLocalizations = lookupZulipLocalizations(ZulipLocalizations.supportedLocales.first);
```

Comment:
Separately, let's put this line just before where we use it. It logically fits as part of checking the end state, vs. preparing the state.